internal/pubsub/pubsub_test.go

```diff
@@ -152,6 +152,7 @@ func TestConcurrentReceivesGetAllTheMessages(t *testing.T) {
 						return
 					}
 					t.Error(err)
```

Comment:
Or `t.Fatal` instead?

Reply:
`t.Fatal` calls `t.FailNow`, and that must not be called from goroutines other than the one running the test. https://golang.org/pkg/testing/#B.FailNow

Reply:
Hmm, I did not know that! Learn something every day. Well, some days.